src/OrchardCore.Themes/TheBlogTheme/Views/Widget-Image.liquid

```diff
@@ -0,0 +1,4 @@
+﻿<article class="{{ Model.Classes | join: " " }}">
```

Comment:
You can delete this file, it was not used. I might add a variation of this one later